### ISSUE TITLE
eshell can't load .envrc via Tramp

### ISSUE THREAD

**Issue by treed** [purcell/envrc#27]
If I try to use 'envrc-allow` in an eshell which is on another machine via Tramp with ssh, direnv is unable to find the .envrc:

> direnv: error .envrc file not found

If I use a copy of this directory locally, it successfully finds .envrc. My guess is that something with the execution of the direnv command isn't wrapping it correctly with TRAMP. I'm vaguely aware that something along these lines was improved with Emacs 27, but I'm on 27.2, so there might be additional work here?

**Comment by purcell** [comment]
Yes, probably something like https://github.com/wbolster/emacs-direnv/pull/68 will be needed, but it will be a bit more complex for `envrc`, I think.
